bugbug/model.py

```diff
@@ -254,8 +255,18 @@ def train(self, importance_cutoff=0.15):
 
         print(f"No confidence threshold - {len(y_test)} classified")
         confusion_matrix = metrics.confusion_matrix(y_test, y_pred, labels=class_names)
-        print(confusion_matrix)
         tracking_metrics["confusion_matrix"] = confusion_matrix.tolist()
+        for i in range(len(tracking_metrics["confusion_matrix"])):
+            tracking_metrics["confusion_matrix"][i].insert(0, class_names[i])
```

Comment:
We shouldn't modify the `tracking_metrics` object

Reply:
Got it